src/app/login/components/LoginForm.jsx

```diff
@@ -1,38 +1,15 @@
 "use client";
 
-import { useRouter } from "next/navigation";
-import React, { useState } from "react";
-import Cookies from "js-cookie";
+import { useRequestLogin } from "@/lib/fetchAPI";
 
 export const LoginForm = () => {
-  const router = useRouter();
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
-
-  async function handleSubmitLogin(e) {
-    e.preventDefault();
-
-    const res = await fetch("https://eventmakers-api.fly.dev/auth/login", {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify({
-        email,
-        password,
-      }),
-    });
-    const { payload, token } = await res.json();
-
-    localStorage.setItem("user", JSON.stringify(payload));
-
-    Cookies.set("token", token);
-    router.push("/dashboard");
-  }
+  const { loading, setEmail, setPassword, handleSubmitLogin } =
+    useRequestLogin();
 
   return (
-    <div className="flex w-full h-full">
-      <div className="bg-blue-500 h-[100vh] w-[50vw]"></div>
+    <div className="flex flex-col items-center justify-center m-auto h-[100vh] gap-10">
+      <h1 className="text-4xl">EVENTMAKER</h1>
+      <h1>LOGIN</h1>
```

Comment:
There can't be more than one `<h1>`. Please change the other one into `<h2>`
Login text should be bigger than Eventmaker.

Thankss